Make the man directory if it's missing before installing the man page.
Already upstream see http://hg.podgorny.cz/unionfs-fuse/rev/6ca07ad1b855

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura unionfs-fuse-0.24.orig/Makefile unionfs-fuse-0.24/Makefile
--- unionfs-fuse-0.24.orig/Makefile	2010-05-20 08:04:08.000000000 -0300
+++ unionfs-fuse-0.24/Makefile	2012-01-11 08:08:02.382667776 -0300
@@ -9,5 +9,6 @@
 
 install: build
 	install -d $(DESTDIR)$(PREFIX)$(BINDIR)
+	install -d $(DESTDIR)$(PREFIX)/share/man/man8
 	install -m 0755 src/unionfs $(DESTDIR)$(PREFIX)$(BINDIR)
 	install -m 0644 man/unionfs-fuse.8 $(DESTDIR)$(PREFIX)/share/man/man8/
